content/blog/2017/2017-04-05-welcome-to-blue-ocean.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Getting Started with Blue Ocean"
+tags:
+- blueocean
+- ux
+- pipeline
+
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:https://cloudbees.com[CloudBees].
+
+== Welcome to Blue Ocean 1.0!
+
+In case you've been heads down on other projects
+link:/blog/2016/05/26/introducing-blue-ocean/[for the past 10 months],
+Blue Ocean is a new user experience for Jenkins,
+and version 1.0 was released today!
+Blue Ocean takes Jenkins beyond of the realm of infrastructure tools used only by specialists,
```

Comment:
"realm of infrastructure tools used only by specialists" comes across really weird to me. I never thought of it that way. I feel like you're trying to convey that Blue Ocean makes Jenkins more approachable to more than just developers.